libs/sequelize.js

```diff
@@ -0,0 +1,13 @@
+const { Sequelize } = require('sequelize');
+const { config } = require('./../config/config');
+
+const USER = encodeURIComponent(config.dbUser);
+const PWD = encodeURIComponent(config.dbPassword);
+const URI = `postgres://${USER}:${PWD}@${config.dbHost}:${config.dbPort}/${config.dbName}`;
```

Comment:
Maybe you can move this to an isolated file because it is the same for `libs/postgres.pool.js` and `libs/postgres.js`.